cmd2/argparse_completer.py

```diff
@@ -1051,7 +1049,7 @@ def format_help(self) -> str:
         formatter.add_text(self.epilog)
 
         # determine help from format above
-        return formatter.format_help()
+        return formatter.format_help() + '\n'
```

Comment:
This is fine, but could make use of `LINE_FEED` constant in [constants.py](https://github.com/python-cmd2/cmd2/blob/master/cmd2/constants.py#L21).

Alternatively, if we don't mind it being `\r\n` on Windows, there is the built-in `os.linesep` constant.

Reply:
We use `\n` and not `\r\n` everywhere else in **cmd2**, so I'm OK not using `os.linesep`.